### PR TITLE
Updating Python hot HTTP/JSON sample

### PR DESCRIPTION
**Pull request by denismakogon**
- using latest fdk 0.0.1 release

### CODE REVIEW

**Comment by denismakogon** [comment]
https://github.com/fnproject/fdk-python/pull/1 <--- slightly modified version of hotfn-py.

**Comment by rdallman** [comment]
@denismakogon take it this not ready since we changed library names and so on, don't see that reflected here. also needs rebase, not sure if part of last patch. want to update or should we close this ?

**Comment by denismakogon** [comment]
Yeah, need to release fdk in first place.

**Comment by denismakogon** [comment]
@treeder samples update with respect to first fdk release.

File: examples/formats/http/python/func.py [hunk 1]
```diff
@@ -1,42 +1,35 @@
...
-        rs = response.RawResponse(
-            http_version, 200, "OK",
-            response_data=req_data)
-        print(rs.dump(), file=sys.stdout, flush=True)
-        super(MyProtocol, self).data_received(data)
-
-    def connection_lost(self, exc):
-        print('pipe closed', file=sys.stderr, flush=True)
-        super(MyProtocol, self).connection_lost(exc)
+# All Rights Reserved.
```
Comment (treeder) [inline examples/formats/http/python/func.py]: Remove license from this file.

File: examples/formats/json/python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM python:3.6.2
```
Comment (treeder) [inline examples/formats/json/python/Dockerfile]: Does it need it's own Dockerfile?  Looks like it's doing the basics that `fn build` already covers.

File: examples/formats/json/python/func.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# All Rights Reserved.
```
Comment (treeder) [inline examples/formats/json/python/func.py]: Remove all of these.

File: examples/formats/json/python/func.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+name: denismakogon/hot-json-python:0.0.1
+version: 0.0.1
+runtime: docker
```
Comment (treeder) [inline examples/formats/json/python/func.yaml]: Why not just python?

File: examples/formats/json/python/func.py (hunk 3 above)
Comment (denismakogon) [reply examples/formats/json/python/func.py]: They are required by apache 2.0 licence, no?

File: examples/formats/json/python/func.yaml (hunk 4 above)
Comment (denismakogon) [reply examples/formats/json/python/func.yaml]: `python3.5.2` or `python3.6.2` to be more precise.

**Pull request merged**